types/sinon-express-mock/index.d.ts

```diff
@@ -1,55 +1,63 @@
 // Type definitions for sinon-express-mock 1.3
 // Project: https://github.com/danawoodman/sinon-express-mock#readme
 // Definitions by: Jared Chapiewsky <https://github.com/jpchip>
+//                 Tomek Łaziuk <https://github.com/tlaziuk>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import * as Sinon from "sinon";
+import {
+    SinonStub,
+} from "sinon";
+
+import {
+    Request,
+    Response,
+} from "express";
 
 export namespace mockReq {
-	interface MockReq {
-		accepts: Sinon.SinonStub;
-		acceptsCharsets: Sinon.SinonStub;
-		acceptsEncodings: Sinon.SinonStub;
-		acceptsLanguages: Sinon.SinonStub;
-		body: object;
-		flash: Sinon.SinonStub;
-		get: Sinon.SinonStub;
-		is: Sinon.SinonStub;
-		params: object;
-		query: object;
-		session: object;
-	}
+    interface MockReq {
+        accepts: SinonStub;
+        acceptsCharsets: SinonStub;
+        acceptsEncodings: SinonStub;
+        acceptsLanguages: SinonStub;
+        body: object;
+        flash: SinonStub;
+        get: SinonStub;
+        is: SinonStub;
+        params: object;
+        query: object;
+        session: object;
+    }
 }
 
 export namespace mockRes {
-	interface MockRes {
-		append: Sinon.SinonStub;
-		attachement: Sinon.SinonStub;
-		clearCookie: Sinon.SinonStub;
-		cookie: Sinon.SinonStub;
-		download: Sinon.SinonStub;
-		end: Sinon.SinonStub;
-		format: object;
-		get: Sinon.SinonStub;
-		headersSent: Sinon.SinonStub;
-		json: Sinon.SinonStub;
-		jsonp: Sinon.SinonStub;
-		links: Sinon.SinonStub;
-		locals: object;
-		location: Sinon.SinonStub;
-		redirect: Sinon.SinonStub;
-		render: Sinon.SinonStub;
-		send: Sinon.SinonStub;
-		sendFile: Sinon.SinonStub;
-		sendStatus: Sinon.SinonStub;
-		set: Sinon.SinonStub;
-		status: Sinon.SinonStub;
-		type: Sinon.SinonStub;
-		vary: Sinon.SinonStub;
-	}
+    interface MockRes {
+        append: SinonStub;
+        attachement: SinonStub;
+        clearCookie: SinonStub;
+        cookie: SinonStub;
+        download: SinonStub;
+        end: SinonStub;
+        format: object;
+        get: SinonStub;
+        headersSent: SinonStub;
+        json: SinonStub;
+        jsonp: SinonStub;
+        links: SinonStub;
+        locals: object;
+        location: SinonStub;
+        redirect: SinonStub;
+        render: SinonStub;
+        send: SinonStub;
+        sendFile: SinonStub;
+        sendStatus: SinonStub;
+        set: SinonStub;
+        status: SinonStub;
+        type: SinonStub;
+        vary: SinonStub;
+    }
 }
 
-export function mockReq(options?: object): mockReq.MockReq;
+export function mockReq<T extends object>(options?: T): mockReq.MockReq & T & Request;
```

Comment:
I don't know if this is the proper way either. Now typings "thinks" there is eg. cookies property in that mock, when there is no such property. Thankfully the cookies is any, but there are also other properties which are not included in that mock. I think you know what I want to say 😊